### ISSUE TITLE
Bug: FAnim not applying correctly

### ISSUE THREAD

**Issue by SolarLune** [GoranM/bdx#165]
Hey, there. So this seems to be a bug with FAnim - it seems like the animation applies changes from whatever the object's transform is in the game scene, rather than starting with the actual frame data in the animation. 

For example, if you were to make an animation where the camera rotates up 90 degrees from looking straight, and play the animation when the camera is already looking up, it would rotate 90 degrees `again` to be upside down now. 

Basically, it doesn't seem like playing an animation sets the position, scale, and rotation of the object, but rather applies delta changes to the object transform. Maybe it would be good to keep this delta / local functionality as an option, and have both?

**Comment by GoranM** [comment]
Yes, it's additive, and that's by design - so, technically, not a bug.

If you want the ability to change that default, you can write an additive(boolean) method, where offsets are added to the base channel value if true ([as it stands now](https://github.com/GoranM/bdx/blob/master/src/com/nilunder/bdx/components/FAnim.java#L185)), or excluded if false.

**Comment by SolarLune** [comment]
OK, I did it. PR submitted.

**Comment by SolarLune** [comment]
PR merged. Closed.
